tests/scalability/csv/person.clinic.csv

```diff
@@ -1,5999 +1,5999 @@
-person_3072,83072,clinic_9,Person 3072,2008-06-25,+16143334626,male,1530979223000
-person_3073,83073,clinic_9,Person 3073,2008-06-26,+16143334627,female,1530979224000
-person_3074,83074,clinic_9,Person 3074,2008-06-27,+16143334628,male,1530979225000
-person_3075,83075,clinic_9,Person 3075,2008-06-28,+16143334629,female,1530979226000
-person_3076,83076,clinic_9,Person 3076,2008-06-29,+16143334630,male,1530979227000
-person_3077,83077,clinic_9,Person 3077,2008-06-30,+16143334631,female,1530979228000
-person_3078,83078,clinic_9,Person 3078,2008-07-01,+16143334632,male,1530979229000
-person_3079,83079,clinic_9,Person 3079,2008-07-02,+16143334633,female,1530979230000
-person_3080,83080,clinic_9,Person 3080,2008-07-03,+16143334634,male,1530979231000
-person_3081,83081,clinic_9,Person 3081,2008-07-04,+16143334635,female,1530979232000
-person_3082,83082,clinic_9,Person 3082,2008-07-05,+16143334636,male,1530979233000
-person_3083,83083,clinic_9,Person 3083,2008-07-06,+16143334637,female,1530979234000
-person_3084,83084,clinic_9,Person 3084,2008-07-07,+16143334638,male,1530979235000
-person_3085,83085,clinic_9,Person 3085,2008-07-08,+16143334639,female,1530979236000
-person_3086,83086,clinic_9,Person 3086,2008-07-09,+16143334640,male,1530979237000
-person_3087,83087,clinic_9,Person 3087,2008-07-10,+16143334641,female,1530979238000
-person_3088,83088,clinic_9,Person 3088,2008-07-11,+16143334642,male,1530979239000
-person_3089,83089,clinic_9,Person 3089,2008-07-12,+16143334643,female,1530979240000
-person_3090,83090,clinic_9,Person 3090,2008-07-13,+16143334644,male,1530979241000
-person_3091,83091,clinic_9,Person 3091,2008-07-14,+16143334645,female,1530979242000
-person_3092,83092,clinic_9,Person 3092,2008-07-15,+16143334646,male,1530979243000
-person_3093,83093,clinic_9,Person 3093,2008-07-16,+16143334647,female,1530979244000
-person_3094,83094,clinic_9,Person 3094,2008-07-17,+16143334648,male,1530979245000
-person_3095,83095,clinic_9,Person 3095,2008-07-18,+16143334649,female,1530979246000
-person_3096,83096,clinic_9,Person 3096,2008-07-19,+16143334650,male,1530979247000
-person_3097,83097,clinic_9,Person 3097,2008-07-20,+16143334651,female,1530979248000
-person_3098,83098,clinic_9,Person 3098,2008-07-21,+16143334652,male,1530979249000
-person_3099,83099,clinic_9,Person 3099,2008-07-22,+16143334653,female,1530979250000
-person_3100,83100,clinic_9,Person 3100,2008-07-23,+16143334654,male,1530979251000
-person_3101,83101,clinic_9,Person 3101,2008-07-24,+16143334655,female,1530979252000
-person_3102,83102,clinic_9,Person 3102,2008-07-25,+16143334656,male,1530979253000
-person_3103,83103,clinic_9,Person 3103,2008-07-26,+16143334657,female,1530979254000
-person_3104,83104,clinic_9,Person 3104,2008-07-27,+16143334658,male,1530979255000
-person_3105,83105,clinic_9,Person 3105,2008-07-28,+16143334659,female,1530979256000
-person_3106,83106,clinic_9,Person 3106,2008-07-29,+16143334660,male,1530979257000
-person_31
```

Comment:
the `person_3997` and `person_3998` are duplicated here so that they're A) not unique and B) out of sync with the next column.  This error was in the prior dataset, so possibly not an issue.

Reply:
Nice catch! This isn't the same duplicate tho.
```
duplicate reference person_3997
duplicate reference person_3998
duplicate reference person_3997
duplicate reference person_3998
duplicate reference person_4995
duplicate reference person_4996
duplicate reference person_4995
duplicate reference person_4996
```

I'm guessing, since this is set as a reference, then the referencing thing will just use the 1st reference and the other will be discarded.

Reply:
This likely would not have presented any issues but I went ahead and removed it. It should not have been there.